modules/custom/rest_entity_recursive/rest_entity_recursive.info.yml

```diff
@@ -0,0 +1,9 @@
+name: REST Entity Recursive
+description: Adds format to REST endpoints. The request with the format includes entity with whole references entities.
```

Comment:
I'd update description to:
A new "json_recursive" REST format exposes all fields and referenced entities by default.

Reply:
Changed.